### DISCUSSION TITLE
Render .vtu file on demand on web Page using Docker

### DISCUSSION THREAD

**Discussion by clabnet** [Kitware/trame#630] [Q&A] [answered]
Hi,  I need to render a .vtu content into my web Page using Trame as IFrame.

The url selected from the user (wich contain the address where the .vtu Is published) is required to pass as parameter to Trame server running on Docker container.

Is required an APi (flask, fastapi) to get the url file and pass to trame server or Is there another simple way ?

Thank you.

**Comment by jourdain** [comment, marked as answer]
https://github.com/Kitware/trame/discussions/518

**Reply by ClaudioBarcaTEORESI** [reply]
Good, it work. 
Thanks @jourdain @chiyandetaotie
